Firebase.agent.lib.nut

```diff
@@ -570,7 +614,7 @@ class Firebase {
     }
 
     // return a Promise if the Promise library is included
-    function _returnPromise (request){
+    function _returnPromise(request) {
```

Comment:
I'm also concerned with the code duplication, which is a very error prune approach as you can see (implementation of the two functions already got out of sync)